### ISSUE TITLE
chat: backtick-fenced code blocks are rendered with whitespace between lines

### ISSUE THREAD

**Issue by urcades** [urbit/urbit#3402]
**Describe the bug**

Refer to this photo:

<img width="323" alt="Screen Shot 2020-08-27 at 11 38 50 PM" src="https://user-images.githubusercontent.com/1195363/91518963-57f08700-e8bf-11ea-8d09-d7e159b7c169.png">

**To Reproduce**
1. Go to the chat module
2. Use the 3-backtick notation to fence a block of text as so, using shift-return to insert newlines:

```
\```
like this (ignore the backslashes) — line one
line two
line three
\```
```

3. The fenced lines of text will appear with whitespace between the lines, as shown in the image above.

**Expected behavior**

The fenced code should appear as a single block, without whitespace between the lines

**Desktop (please complete the following information):**
 - macOS 10.15.5 (19F101)
 - Chrome 84.0.4147.135 (Official Build) (64-bit)
 - [%base-hash 0vo.j1ero.slhgf.9ei39.khjic.7n0kh.6dvfk.f5qas.olpv8.ct3ei.2inqt]

**Comment by matildepark** [comment]
seems to be just a change to use

`code:not(pre code), pre` instead of `code, pre.code`. need to test with inline code blocks / code messages too

**Comment by matildepark** [comment]
Fixed in #3422.
